project_one/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Home Page</title>
+	<link rel="stylesheet" type="text/css" href="index.css">
+	<nav id= "Navigation">
+		<a href="pageTwo.html" id= "secondPage"> Page 2 </a> 
+		<a href="pageThree.html" id= "thirdPage"> Page 3</a>
+		<a href="pageFour.html" id = "fourthPage"> Page 4 </a>
+	</nav>
+
+	<img src="Kappa_img.jpeg" id = "Main_img"> 
+</head>
+<body>
+
+	<script type="text/javascript" src= "index.js"></script>
+
+	<h1 id = "title"> WELCOME TO MY FIRST PROJECT </h1>
+
+	<div id= "First_Part">
+		<p id = "paragraph_one"> This is section one <button id = "grading-element-one"  onclick = "elementOne()"> Add Something </button> </p> 
+
+	</div>
+
+	<div id= "Second_Part">
+		<a href= "about.html" id= "grading-element-two" target = "_blank" > About Page </a>
+		
+	</div>
+
+	<div id = "Third_Part">
+
+		<br></br>
+	<p> Once YOU CLICK YOU'RE DONE ... <button id = "grading-element-three" onclick = "elementThree()" > Click Here FOR MAGIC </button>
+	</p>
+
+
+	</div>
+
+	<div id = "Fourth_Part">
+
+	 <form name = "grading-element-four" method = "POST" action ="http://localhost:3000/form">
```

Comment:
`document.querySelector('form[name="grading-element-four"]).click()' does not submit the form...

Don't get me wrong, validation is great! But in this case i need the values to be pre-populated by your javascript...